src/level_four/core/tasks/send_transaction_tasks_eth.c

```diff
@@ -121,6 +121,26 @@ void send_transaction_tasks_eth()
         address_scr_init(top_heading, display, true);
     } break;
 
+    case SEND_TXN_VERIFY_BLIND_SIGNING_ETH: {
+        if (eth_unsigned_txn_ptr.payload_status == PAYLOAD_SIGNATURE_NOT_WHITELISTED) {
+            char display[125] = {0};
+            instruction_scr_destructor();
+            snprintf(display, sizeof(display), "%s Blind Signing\nProceed at your own risk!", LV_SYMBOL_WARNING);
+            confirm_scr_init(display);
+        } else {
+            mark_event_over();
+        }
+    } break;
+
+    case SEND_TXN_VERIFY_DERIVATION_PATH:{
+            char display[125] = {0};
+            char path[128] = {0};
+            eth_derivation_path_to_string(&var_send_transaction_data.transaction_metadata,path,sizeof(path));
+            instruction_scr_destructor();
+            snprintf(display, sizeof(display), "Verify Derivation Path\n%s",path);
+            confirm_scr_init(display);
```

Comment:
when do we have to show derivation path?

currently its SHOWN IN ALL transactions except for the case of UNVERIFIED token transfer.